Add blog post for entity resolution with Senzing

prrao87 opened the pull request:
This PR introduces a blog post that describes the [workshop at KGC 2025](https://github.com/kuzudb/kgc-2025-workshop-high-quality-graphs) organized by Kuzu and Senzing. The aims of the post are the following:

- Throw a spotlight on the two data sources (OpenSanctions and Open Ownership) that provide data about financial crimes
- Illustrate how to run a Senzing entity resolution workflow on the data from these two sources to generate an "overlay" that can help connect the disjoint subgraphs
- Demonstrate the Kuzu data transformation and ingestion pipeline
- Show example Cypher queries to investigate the graph
- Show an example network analysis that helps uncover deeper insights

@ceteri would love to have your thoughts on the way I've described the Senzing workflow, as well as the OpenSanctions/Open Ownership datasets. If you think we can add more descriptions about the data and how to do more with it, please feel free to add footnotes where appropriate. I think the main blog should be concise where possible, but we can add more information via footnotes (as I've done in a few cases). Thanks!

You can clone the repo and run a local server on your end to see the blog locally (it requires npm):
```bash
npm i
npm start
```

ceteri commented on public/img/creating-high-quality-knowledge-graphs/kgc-er-3.png:
How about, can we add a looped edge from Sz Entity back to Sz entity?

We often have some entities linking to "related entities" where the probability is not high enough to cause a "merge" decision -- but these are great "leads" in investigations

ceteri commented:
Also, I wanted to add: the first few paragraphs provide such a clear, concise description of how and why one would want to use entity resolution when putting together data for a building a knowledge graph. Brilliant!

prrao87 commented:
> I've added the one comment about the schema diagram showing Sz entities -- if they could link to each other? Though we may not have such cases in these data slices, it'd probably be a good point to illustrate since that happens so often in investigative cases.
I've addressed this here: https://github.com/kuzudb/kuzu-blog/pull/122

Thanks for the review!
